asn.go

```diff
@@ -71,3 +78,22 @@ func NewAsnStore() (*asnStore, error) {
 
 	return &asnStore{cr}, nil
 }
+
+type indirectAsnStore struct {
+	store *asnStore
+}
+
+// AsnForIPv6 returns the AS number for the given IPv6 address.
+// If no mapping exists for the given IP, this function will
+// return an empty ASN and a nil error.
```

Comment:
is there a default cidr prefix for unknown mappings?

Reply:
Can you elaborate a little more, I'm not understanding the question

Reply:
if 2 ips in the same /64 are looked up in a space that isn't in the mapping, will they get the same empty ASN, or different ones?
when we make a new ASN in response to a query, we should maybe save it in the mapping with some default prefix so that other IPs close to it cluster with it.

Reply:
> will they get the same empty ASN, or different ones?

We return `""` and when this is used in kbucket we just reject ASNs named `""`. Note, this is why we're going to have to change your test in go-ipfs to use a real ASN https://github.com/ipfs/go-ipfs/blob/fe9f49aeb71c753b92ed41922c493ff3bf08ef56/test/integration/wan_lan_dht_test.go#L54